bundles/org.openhab.binding.ferroamp/src/main/resources/OH-INF/thing/Channels.xml

```diff
@@ -0,0 +1,978 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="ferroamp"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+	<channel-type id="ehub-wloadconsq-l1">
+		<item-type>Number:Energy</item-type>
+		<label>EHUB, WLoadConsq L1</label>
```

Comment:
All channels have the label prefix EHUB, this can be removed as it adds no value.

Reply:
Please, see commit 20240607